testnet/stacks-node/src/event_dispatcher.rs

```diff
@@ -74,6 +77,21 @@ pub const PATH_ATTACHMENT_PROCESSED: &str = "attachments/new";
 pub const PATH_PROPOSAL_RESPONSE: &str = "proposal_response";
 pub const PATH_POX_ANCHOR: &str = "new_pox_set";
 
+lazy_static! {
+    pub static ref STACKER_DB_CHANNEL: StackerDBChannel = StackerDBChannel::new();
```

Comment:
This channel is how the miner thread receives StackerDB events. The channel only gets events if the channel is actually held by a thread.